### DISCUSSION TITLE
Default EOL

### DISCUSSION THREAD

**Discussion by Aleksey-Danchin** [suren-atoyan/monaco-react#247] [Q&A]
Hi! How to set default EOL? setEOL is method of ITextModel. But how to set CRLF mode globaly? I have LF, but needed CRLF.

**Comment by suren-atoyan** [comment]
you can find it in `monaco.editor.EndOfLinePreference.CRLF`, here is the corresponding [doc link](). also, check [this](https://codesandbox.io/s/push-eol-6ye4t)

**Reply by cirosantilli** [reply]
I'm unable to understand the behavoiur of setEOL: https://github.com/microsoft/monaco-editor/issues/3440 or it is buggy.
